common/app/model/FaciaDisplayElement.scala

```diff
@@ -23,6 +24,23 @@ object FaciaDisplayElement {
       case _ => InlineImage.fromFaciaContent(faciaContent)
     }
   }
+
+  def fromContent(apiContent: contentapi.Content): Option[FaciaDisplayElement] = {
+    val maybeEndSlateComponents = for {
+      sectionId <- apiContent.sectionId
+      fields <- apiContent.fields
+      shortUrl <- fields.shortUrl
+    } yield EndSlateComponents(None, sectionId, shortUrl)
+
+    val elements = Elements.make(apiContent)
+
+    maybeEndSlateComponents flatMap { endSlateComponents =>
+      elements.mainVideo match {
+        case Some(videoElement) => Option(InlineVideo(videoElement, apiContent.webTitle, endSlateComponents.toUriPath, Option(InlineImage(videoElement.images))))
```

Comment:
I find this line a little hard to read, do you mind separating onto multiple lines?